### PR TITLE
Inline badges

### PR DESCRIPTION
**Pull request by XhmikosR**
@mozfreddyb: I have added 2 patches with different approaches. Let me know what you prefer.

The idea is that with this we reduce the requests and generally since the SVGs are small, this should be better.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{{! This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at https://mozilla.org/MPL/2.0/. }}
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
```
Comment (mozfreddyb) [inline templates/index.html]: Do you think we should send this to everyone requesting our web page? I'd prefer a template comment.

**Comment by mozfreddyb** [comment]
We're reducing the amount of requests, but the bytes will be more given that you'll now always download fail _and_ pass. Right?

**Comment by XhmikosR** [comment]
I believe the total bytes is less with this patch.

File: templates/index.html (hunk 1 above)
Comment (XhmikosR) [reply templates/index.html]: I don't remember why I changed the comments... but I must had trouble with them.

I'm gonna revisit this patch later.

**Comment by XhmikosR** [comment]
Here's the result locally

master:

```
9 requests
12.32 KB / 11.95 KB transferred
Finish: 399 ms
DOMContentLoaded: 51 ms
load: 301 ms
```

this:

```
7 requests
12.02 KB / 10.46 KB transferred
Finish: 311 ms
DOMContentLoaded: 48 ms
load: 167 ms
```

I'm just not sure

1. which solution we should go with
2. I need to check out the comments before merging this

File: templates/index.html (hunk 1 above)
Comment (XhmikosR) [reply templates/index.html]: I remember why I switched this to a comment. Because I removed the comment from the SVG files.

**Comment by mozfreddyb** [comment]
Thanks for the measurements. I'm convinced :)

**Pull request merged**